### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by nikcode9**
typos.

### CODE REVIEW

**Review by russoz — APPROVED** [review]
LGTM

File: README.rst [hunk 1]
```diff
@@ -103,7 +103,7 @@ Although this document focuses on Community (upstream) content, there will be Pr
 Documentation
 ==============
 
-* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_.
```
Comment (Andersson007) [inline README.rst:106]:
```suggestion
* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_.
```
Should we make the other lines in this section contain periods as well?

**Comment by Andersson007** [comment]
> i think it is better to have periods at end of each line here.

Could you please add them?

**Comment by gotmax23** [comment]
Thanks for the contribution, @nikcode9. Can you explain more on your reasoning for this change? I don't think adding periods to these sentence fragments is correct.

**Comment by Andersson007** [comment]
> Thanks for the contribution, @nikcode9. Can you explain more on your reasoning for this change? I don't think adding periods to these sentence fragments is correct.

@gotmax23 do you mean all or some of the changed sentences? If some, could you please specify them and why you think it's  incorrect? To me it seems if a statement starts with a capital letter it should end with a period when it ends. I'm not an expert though.

File: README.rst [hunk 2]
```diff
@@ -103,9 +103,9 @@ Although this document focuses on Community (upstream) content, there will be Pr
 Documentation
 ==============
 
-* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
-* `Developing Collections <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_
-* `Ansible Collections Requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html>`_
+* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_.
```
Comment (gotmax23) [inline README.rst:106]: None of these should have periods at the end.

File: README.rst [hunk 3]
```diff
@@ -32,27 +32,27 @@ Previously: Ansible 2.9 and earlier
 **Classic Ansible**
 
 * Single repository `ansible/ansible <https://github.com/ansible/ansible>`_.
-* Single package called `ansible`
-* `ansible` had major releases twice a year
-* New features go into the next major release - ie worst case you need to wait 6 months
+* Single package called `ansible`.
+* `ansible` had major releases twice a year.
```
Comment (gotmax23) [inline README.rst:36]:
```suggestion
* Single package called `ansible`
* `ansible` had major releases twice a year
```

File: README.rst [hunk 4]
```diff
@@ -32,27 +32,27 @@ Previously: Ansible 2.9 and earlier
 **Classic Ansible**
 
 * Single repository `ansible/ansible <https://github.com/ansible/ansible>`_.
```
Comment (gotmax23) [inline README.rst:34]:
```suggestion
* Single repository `ansible/ansible <https://github.com/ansible/ansible>`_
```

File: README.rst [hunk 5]
```diff
@@ -32,27 +32,27 @@ Previously: Ansible 2.9 and earlier
...
 
 * The ``ansible/ansible`` (`ansible-base`) repository only contains:
 
-  * The core Ansible programs, ``ansible-{playbook,galaxy,doc,test,etc}``
-  * Some documentation
-  * A tiny subset of modules and plugins to allow for a functioning controller
-  * Together this will be known as ``ansible-base``
+  * The core Ansible programs, ``ansible-{playbook,galaxy,doc,test,etc}``.
+  * Some documentation.
+  * A tiny subset of modules and plugins to allow for a functioning controller.
```
Comment (gotmax23) [inline README.rst:47]:
```suggestion
  * The core Ansible programs, ``ansible-{playbook,galaxy,doc,test,etc}``
  * Some documentation
  * A tiny subset of modules and plugins to allow for a functioning controller
```

File: README.rst [hunk 6]
```diff
@@ -32,27 +32,27 @@ Previously: Ansible 2.9 and earlier
 **Classic Ansible**
 
 * Single repository `ansible/ansible <https://github.com/ansible/ansible>`_.
-* Single package called `ansible`
-* `ansible` had major releases twice a year
-* New features go into the next major release - ie worst case you need to wait 6 months
+* Single package called `ansible`.
+* `ansible` had major releases twice a year.
+* New features go into the next major release - ie worst case you need to wait 6 months.
```
Comment (gotmax23) [inline README.rst:37]:
```suggestion
* New features go into the next major release, i.e., in the worst case, you need to wait 6 months.
```

**Review by gotmax23 — CHANGES_REQUESTED** [review]

File: README.rst [hunk 7]
```diff
@@ -32,27 +32,27 @@ Previously: Ansible 2.9 and earlier
...
+  * A tiny subset of modules and plugins to allow for a functioning controller.
+  * Together this will be known as ``ansible-base``.
 * The rest of the modules and plugins have been moved into various "collections"
 
   * Ansible Collections:
 
     * Can be released independently of ansible-base and Ansible, at whatever release cycle/cadence the collection maintainer prefers.
-    * Will have their own repo (GitHub, GitLab, etc) with dedicated backlog, ie no more shared massive issue & PR backlog
-    * Should still have CI testing and in many cases can be tested more thoroughly
+    * Will have their own repo (GitHub, GitLab, etc) with dedicated backlog, ie no more shared massive issue & PR backlog.
```
Comment (russoz) [inline README.rst:54]:
```suggestion
    * Will have their own repo (GitHub, GitLab, etc) with dedicated backlog, i.e. no more shared massive issue & PR backlog.
```

**Comment by Andersson007** [comment]
@gotmax23 @russoz thanks!

@nikcode9 could you please take a look and press "Commit suggestion" where you agree and express your concerns where you disagree?

**Comment by Andersson007** [comment]
@nikcode9 thanks!

@gotmax23 @russoz please take another look

**Review by Andersson007 — APPROVED** [review]

**Review by gotmax23 — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (gotmax23) [reply README.rst]: https://github.com/ansible-collections/overview/pull/229/commits/a92311160d7778b034aa3bee9c03734d10e60656

**Comment by Andersson007** [comment]
@nikcode9 thanks for the contribution!
@russoz @gotmax23 thanks for reviewing and merging!

**Pull request merged**